docs/reference/concepts/snapshots.md

```diff
@@ -2,32 +2,52 @@
 
 # Snapshots
 
-_This document relates to both [MonoBehaviour and ECS workflows]({{urlRoot}}/reference/workflows/overview)._
+A [snapshot]({{urlRoot}}/reference/glossary#snapshot) is a representation of the state of a simulated world at some point in time. It stores each [persistent]({{urlRoot}}/reference/glossary#persistence) [entity]({{urlRoot}}/reference/glossary#spatialos-entity) and the values of their [components’]({{urlRoot}}/reference/glossary#spatialos-component) properties.
 
-A [snapshot](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#snapshot) is a representation of the state of a simulated world at some point in time. It stores each [entity](https://docs.improbable.io/reference/13.2/shared/glossary#entity) (as long as the entity has the [Persistence component](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#persistence)) and the values of the entity’s [components’](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#component) [properties](https://docs.improbable.io/reference/13.2/shared/glossary#property).
-
-You use a snapshot as the starting point for your [world](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#spatialos-world) when you [deploy](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#deploying), [locally](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#local-deployment) or [to the cloud](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#cloud-deployment).
+You use a snapshot as the starting point for your [world]({{urlRoot}}/reference/glossary#spatialos-world) when you [deploy]({{urlRoot}}/reference/glossary#deploying).
```

Comment:
```suggestion
You can use a snapshot as the starting point for your [world]({{urlRoot}}/reference/glossary#spatialos-world) when you [deploy]({{urlRoot}}/reference/glossary#deploying).
```

Maybe? to highlight that we don't have to pass in a snapshot to start a cloud dpl?

Reply:
It actually _really_ depends how you start it 😬